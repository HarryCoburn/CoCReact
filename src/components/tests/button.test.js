import React from "react";
import renderer from "react-test-renderer";
import Btn from "../button";

const dummyFunc = () => {};
const dummy = {
  label: "test label",
  toolTip: "test toolTip",
  toolTipPos: "top"
};

describe("Btn Component", () => {
  test("Btn rendered correctly", () => {
    const component = renderer.create(
      <Btn
        label={dummy.label}
        toolTip={dummy.toolTip}
        toolTipPos={dummy.toolTipPos}
        onClick={dummyFunc}
      ></Btn>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
