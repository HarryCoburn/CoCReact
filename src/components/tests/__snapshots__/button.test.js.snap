// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Btn Component Btn rendered correctly 1`] = `
<button
  className="button tooltip"
  onClick={[Function]}
>
  test label
  <span
    className="tooltiptext top"
  >
    test toolTip
  </span>
</button>
`;
